app/Models/V2/Projects/Project.php

```diff
@@ -131,11 +131,7 @@ class Project extends Model implements MediaModel, AuditableContract, EntityMode
         'pct_employees_marginalised',
         'pct_beneficiaries_marginalised',
         'pct_beneficiaries_men',
-        'proposed_gov_partners',
-        'proposed_num_nurseries',
-        'proj_boundary',
-        'states',
-        'proj_impact_biodiv',
```

Comment:
I'm confused why this is still showing that you're removing these lines. If this branch and the branch you're merging into are both updated from staging, there should be no diff here at all. These properties are needed, so I'm a bit worried that the changes you've made will cause them to get removed on `staging` once this is all merged in.

Reply:
I got confused too with previous comment. But confirmed that these fields are on the target branch. Nonetheless to prevent deletion of these fields, I am returning them back, in this way the values will remain.